codec-http/src/main/java/io/netty/handler/codec/http/HttpHeaders.java

```diff
@@ -1693,4 +1693,12 @@ public boolean contains(CharSequence name, CharSequence value, boolean ignoreCas
     public String toString() {
         return HeadersUtils.toString(getClass(), iteratorCharSequence(), size());
     }
+
+    /**
+     * Returns a deap copy of the passed in {@link HttpHeaders}.
+     */
+    public static HttpHeaders copy(HttpHeaders headers) {
```

Comment:
can we just make `copy()` a method of the `HttpHeaders` class because it is an abstract class? Then we don't have to use the static method outside and can just call `.copy()` on the object.